python/whylogs/core/metrics/metrics.py

```diff
@@ -40,6 +40,7 @@ class MetricConfig:
     kll_k: int = 256
     fi_lg_max_k: int = 10  # 128 entries
     fi_disabled: bool = False
+    track_unicode_ranges: bool = True
```

Comment:
Do we have some quantifiable measurements on how much perf / memory footprint on the profiling step?

Reply:
Good call out to hold off on enabling by default before we have some better numbers. Splitting up this change:
1) Fixes to allow enabling unicode_range tracker turned on as part of profiling (keeping this in this CR)

2) Enabling unicode_range tracker by default (rolled back for now, to limit risk pending better understanding of impact after some benchmarking).